src/soep_preparation/clean_variables/pl.py

```diff
@@ -111,17 +111,17 @@ def clean(raw_data: pd.DataFrame) -> pd.DataFrame:
         renaming={"[2] Nein": False, "[1] Ja": True},
         ordered=True,
     )
-    out["mutterschaftsgeld_bezug_aktuell"] = object_to_bool_categorical(
+    out["bezieht_aktuell_mutterschaftsgeld"] = object_to_bool_categorical(
         series=raw_data["plc0152_v1"],
         renaming={"[1] Ja": True},
     )
-    out["mutterschaftsgeld_brutto_betrag_m_aktuell"] = object_to_float(
+    out["im_letzten_monat_erhaltenes_mutterschaftsgeld_m"] = object_to_float(
         raw_data["plc0153_h"]
     )
-    out["mutterschaftsgeld_betrag_durchschnittlich_m"] = object_to_float(
+    out["durchschnittlich_erhaltenes_mutterschaftsgeld_m"] = object_to_float(
         raw_data["plc0155_h"]
     )
-    out["child_alimony_m"] = object_to_int(raw_data["plc0178"])
+    out["kindesunterhalt_m"] = object_to_int(raw_data["plc0178"])
```

Comment:
```suggestion
    out["kindesunterhalt_erhalten_m"] = object_to_int(raw_data["plc0178"])
```
Same here, I guess? (I know it is inconsistent with the naming just before, but it's just a pain with those different periods...)

However, this now appears here and in `pequiv` (`_y` version there). If it is the same reference period, maybe combine the two?

Reply:
Sure, I can combine the two! Will convert the second to `m`.